packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButton.tsx

```diff
@@ -272,7 +282,7 @@ export const useSpinButton_unstable = (props: SpinButtonProps, ref: React.Ref<HT
   state.input.value = textValue;
   state.input['aria-valuemin'] = min;
   state.input['aria-valuemax'] = max;
-  state.input['aria-valuenow'] = currentValue;
+  state.input['aria-valuenow'] = currentValue === null ? undefined : currentValue;
```

Comment:
minor nit: although it seems weird, could this be shortened to `currentValue ?? undefined`?

Reply:
Maybe? I would have sworn that `null` would render `aria-valuenow=""` but that does not seem to be the case :)